.ci/ci-setup.sh

```diff
@@ -57,19 +57,52 @@ export CC=$(which "$CC")
 gnome_dl=https://download.gnome.org/sources
 
 # Install required dependencies to build
-# glib, json-glib, libmnl-dev check and cc-oci-runtime
-sudo apt-get -qq install valgrind lcov uuid-dev pkg-config \
-  zlib1g-dev libffi-dev gettext libpcre3-dev cppcheck \
-  libmnl-dev moreutils
+# glib, json-glib, libmnl-dev, check, gcc, cc-oci-runtime and qemu-lite
+
+pkgs=""
+
+# general
+pkgs+=" pkg-config"
+pkgs+=" gettext"
+pkgs+=" rpm2cpio"
+pkgs+=" valgrind"
+
+# runtime dependencies
+pkgs+=" uuid-dev"
+pkgs+=" cppcheck"
+pkgs+=" libmnl-dev"
+pkgs+=" libffi-dev"
+pkgs+=" libpcre3-dev"
+
+# runtime + qemu-lite
+pkgs+=" zlib1g-dev"
+
+# qemu-lite
+pkgs+=" libpixman-1-dev"
+
+# gcc
+pkgs+=" libcap-ng-dev"
+pkgs+=" libgmp-dev"
+pkgs+=" libmpfr-dev"
+pkgs+=" libmpc-dev"
+
+# code coverage
+pkgs+=" lcov"
+
+# chronic(1)
+pkgs+=" moreutils"
+
+eval sudo apt-get -qq install "$pkgs"
 
 function compile {
 	name="$1"
 	tarball="$2"
 	directory="$3"
+	configure_opts="$4"
 
 	chronic tar -xvf ${tarball}
 	pushd ${directory}
-	chronic	./configure --disable-silent-rules
+	eval chronic ./configure --disable-silent-rules "$configure_opts"
```

Comment:
I'm really not sure about all those evals, why not just use $configure_opts. Quoting strikes back again :)